### PR TITLE
ci, Enable ci tests for simplify-networking branch

### PR DESCRIPTION
**Pull request by RamLavi**
Signed-off-by: Ram Lavi <ralavi@redhat.com>

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@ name: CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [ simplify-networking ]
```
Comment (phoracek) [inline .github/workflows/ci.yml:5]: Does not this disable tests for main?

Comment (RamLavi) [reply .github/workflows/ci.yml:5]: you are right! good catch, I'll add it instead of replace it

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -2,9 +2,13 @@ name: CI
...
   push:
-    branches: [ main ]
+    branches:
+      - main
+      - simplify-networking
   pull_request:
-    branches: [ main ]
+    branches:
+       - main
+      - simplify-networking
```
Comment (phoracek) [inline .github/workflows/ci.yml:11]:
```suggestion
    branches:
      - main
      - simplify-networking
```

Comment (RamLavi) [reply .github/workflows/ci.yml]: thanks!

**Pull request merged**